File: java/app/src/main/java/com/jossy/android/mobilebenchmarkappjava/models/CpuResult.java
package com.jossy.android.mobilebenchmarkappjava.data;

public class CpuResult {
        public final int threads;
        public final long durationMs;
        public final long iterations;
        public final double checksum;
        public CpuResult(int threads, long durationMs, long iterations, double checksum) {
            this.threads = threads;
            this.durationMs = durationMs;
            this.iterations = iterations;
            this.checksum = checksum;
        }
    }

File: flutter/lib/models/cpu_result.dart
/// Encapsulates CPU benchmark execution result.
/// Contains metrics for cross-platform comparison.
class CpuResult {
  final int threads;
  final int durationMs;
  final int iterations;
  final double checksum;

  const CpuResult({
    required this.threads,
    required this.durationMs,
    required this.iterations,
    required this.checksum,
  });
}